CHANGELOG.md

```diff
@@ -35,6 +38,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 - kirderf1, hadean, Estellairon, Riotmode, Carnie, Akisephila, Zepsun, Vinnelli, Vivian Zane, Dweblenod
 
+### Fan Contributions
+
+- gummyYummy
+
```

Comment:
Well of course it could just be "Idea Contributions", but I think we should go with something else, but im not sure what. For now that is what I will put, but we have like 12 days to come up with something better.